lsp-mode.el

```diff
@@ -2203,17 +2208,18 @@ PATH is the current folder to be checked."
 
 (defun lsp--headerline-breadcrumb-build-path-up-to-project-string ()
   "Build the path-up-to-project segment string for the breadcrumb."
-  (when (member 'path-up-to-project lsp-headerline-breadcrumb-segments)
-    (seq-reduce (lambda (last-dirs next-dir)
-                  (format "%s%s %s"
-                          (if last-dirs
-                              (concat last-dirs " ")
-                            (if (eq (cl-first lsp-headerline-breadcrumb-segments) 'path-up-to-project)
-                                ""
-                              " "))
-                          (lsp--headerline-breadcrumb-arrow-icon)
-                          (propertize next-dir 'font-lock-face lsp-headerline-breadcrumb-prefix-face)))
-                (lsp--headerline-path-up-to-project-root (lsp-workspace-root) (f-parent (buffer-file-name))) nil)))
+  (when (and (member 'path-up-to-project lsp-headerline-breadcrumb-segments))
```

Comment:
The `and` shouldn't be required?

Reply:
Yeah, I'll remove this, thanks!

Reply:
Done here: https://github.com/emacs-lsp/lsp-mode/pull/1913